Automattic/studio
# Implement Sync connected sites UI

- Closes https://github.com/Automattic/dotcom-forge/issues/9393

## Proposed Changes

- Add connected sites view with cool scroll
- Add tests

## Testing Instructions



- Run `STUDIO_SITE_SYNC=true npm start`
- Click on Sync tab
- Press connect
- select a site
- Connect a site
- Observe the UI is updated and it shows your connected site. If it has staging sites they are appended too.
- Add more sites
- Observe the nice scroll
- Disconnect all sites
- Observe the first screen is displayed


https://github.com/user-attachments/assets/1933c4f5-e851-4ed1-935a-3bf6ce6b7b98



## Pre-merge Checklist
